grails-app/views/wps/_asyncRequest.xml.gsp

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<wps:Execute service="WPS" version="1.0.0"
+  xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:ows="http://www.opengis.net/ows/1.1"
+  xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd">
+
+
+  <ows:Identifier>gs:NetcdfOutput</ows:Identifier>
+
+  <wps:DataInputs>
+    <wps:Input>
+      <ows:Identifier>namespace</ows:Identifier>
+      <wps:Data>
+        <wps:LiteralData>imos</wps:LiteralData>
+      </wps:Data>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>typeName</ows:Identifier>
+      <wps:Data>
+        <wps:LiteralData>${typeName}</wps:LiteralData>
+      </wps:Data>
+    </wps:Input>
+
+    <g:if test="${cqlFilter}">
```

Comment:
I thought here instead, we could iterate on the parameters that the user provided, apart from email, which is mandatory. So same goes to `typeName` and `namespace`. So the using controller passes to the view email + a map of parameters and the view encodes them one by one without asking questions. This makes the portal dumb about WPS - and it should be, with the logic and flexibility kept on JS side, which is where all the download logic resides.